app/views/pages/information.html.erb

```diff
@@ -30,26 +30,20 @@
     filter: invert(0);
     filter: var(--theme-social-icon-invert, invert(0));
   }
-
-  .social-links img {
-    width: 35px;
-    height: 35px;
-    margin: 5px 10px;
-  }
 </style>
 
 <div class="blank-space"></div>
 <div class="container article">
   <div class="title">
     <h1>
-      More information about dev.to(👋)
+      More information about <%= community_name %> 👋
```

Comment:
Took out the brackets here, it looked nicer, but if it's more of a consistent practice I can put the () back